light/world/utils/json_utils.py

```diff
@@ -73,11 +79,13 @@ def convert_dict_to_node(obj, world):
         return obj
 
 
-def read_event_logs(event_file):
+def read_event_logs(event_file: str) -> List[Tuple[str, str, float, str]]:
     """
     Parses an event file, returning the read file in a buffer
     """
     buffer = []
+    world_hash = [None, "None"]
```

Comment:
Looking at the rest of the code, `world_hash` seems to be overwritten in the first run of the loop because of  `parity = 0` in L#90.

Reply:
Indeed, these are just to satisfy the linter